src/search/containers/CurrentSearchPage.js

```diff
@@ -0,0 +1,35 @@
+import { connect } from 'react-redux';
+import SearchPage from '../components/SearchPage';
+import { setQuery } from '../actions';
+
+function getVisibleAddons(addons, query) {
+  const matches = [];
+  for (const slug in addons) {
+    if (addons[slug].title.indexOf(query) >= 0) {
```

Comment:
Apparently this always returns `-1` if it isn't found but it still feels to me like it could return any negative number (I know that isn't the case but it feels wrong to check `-1` explicitly). My reasoning here is that anything `0` or larger is in the array so it was found, `> -1` doesn't seem more direct to me and doesn't pass my mental "any negative number" check because who know, maybe Chakra decides to return `-0.3333333333333...`.